stack.yaml

```diff
@@ -211,7 +211,7 @@ packages:
   extra-dep: true
 
 nix:
-  shell-file: shell.nix
+  shell-file: nix/stack-shell.nix
```

Comment:
does this mean stack works on nixos now?

Reply:
It always did work. The problem was that the nix-shell environments for cabal new-build and stack were incompatible. Since stack runs the nix-shell behind the scenes, we can rename the shell file used for stack, and have the top-level shell.nix compatible with cabal.